app/controllers/api/v1/invitations_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::InvitationsController < Api::V1::ApiController
+  def create
+  	token = SecureRandom.hex(10)
+  	WebsiteMember.create(:website_id => params[:member][:website][:id], :invitation_token => token, :invitation_sent_at => Time.now.utc )
+  	UserMailer.member_invitation(params[:member][:website], params[:member][:email], token, current_member).deliver_now
```

Comment:
ar trebuii sa folosesti strong_parameters aici. Adica sa faci metoda jos `member_params` si sa apelezi aici `member_params[:website]` .... Trimiti website cu totul in params?? Nu prea e bine asta ar trebuii sa trimiti doar website_id probabil de asta ai si facut prostia aia mai sus si nu ai folosit current_website. Trimite website_id cand faci requestul.